cities/praha/venues/remote.yaml

```diff
@@ -0,0 +1,8 @@
+city: Prague
+name: Pouze ONLINE
+address: The Internet (online event only)
+notes: |
+   Událost se uskuteční pouze online. Informace o možnosti připojení budou součástí popisu události.
+location:
+  latitude: '50.0875464'
+  longitude: '14.4212444'
```

Comment:
Old town square?

Reply:
There has to be something. This is copied from `praha.yaml`